allowed.py

```diff
@@ -558,6 +559,10 @@ def check_tree(tree: ast.AST, constructs: tuple, source: list) -> list:
             del errors[index]
     return errors
 
+def is_filetype_supported(filename: str) -> bool:
```

Comment:
The code can be simplified by omitting this function. See my later comment.

Reply:
Removed.